specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -0,0 +1,1107 @@
+}
+
+// @doc("Optional load test config")
+// model OptionalLoadTestConfig {
+//   @doc("""
+// Test URL. Provide the complete HTTP URL. For example,
+// http://contoso-app.azurewebsites.net/login
+// """)
+//   endpointUrl?: string;
+
+//   @doc("No of concurrent virtual users")
+//   virtualUsers?: int32;
+
+//   @added(APIVersions.v2023_04_01_preview)
+//   @doc("Throughput required (requests per second)")
+//   rps?: int32;
+
+//   @added(APIVersions.v2023_04_01_preview)
+//   @doc("Maximum response time of the API/endpoint")
+//   maxResponseTime?: int32;
+
+//   @doc("Ramp up time")
+//   rampUpTime?: int32;
+
+//   @added(APIVersions.v2023_04_01_preview)
+//   @doc("Number of steps in the ramp - up phase")
+//   rampUpSteps?: int32;
+
+//   @added(APIVersions.v2023_04_01_preview)
+//   @doc("Time to maintain a higher load after a spike in traffic")
+//   spikeHoldLoad?: int32;
+
+//   @added(APIVersions.v2023_04_01_preview)
+//   @doc("Multiplier applied to the base load during a spike in traffic")
+//   spikeMultiplier?: float32;
+
+//   @added(APIVersions.v2023_04_01_preview)
+//   @doc("Type of load. e.g. Linear, Constant, StepUp, Spike")
+//   loadType?: LoadType;
+
+//   @added(APIVersions.v2023_04_01_preview)
+//   @doc("Type of load. e.g. Linear, Constant, StepUp, Spike")
+//   testType?: TestType;
+
+//   @doc("Test run duration")
+//   duration?: int32;
+// }
+
+
+
+@doc("The input artifacts for the test.")
+model TestInputArtifacts {
+  @doc("File info")
+  configFileInfo?: FileInfo;
+
+  @doc("File info")
+  testScriptFileInfo?: FileInfo;
+ 
+  @doc("File info")
+  userPropFileInfo?: FileInfo;
+
+  @doc("File info")
+  inputArtifactsZipFileInfo?: FileInfo;
+
+  @added(APIVersions.v2023_04_01_preview)
+  @doc("The config json file for url based test")
+  urlTestsConfigFileInfo?: FileInfo;
+
+  @doc("Additional supported files for the test run")
+  @visibility("read")
+  additionalFileInfo?: FileInfo[];
+}
+
+@doc("File info")
+model FileInfo {
+  @doc("File URL.")
+  url?: string;
+
+  @doc("Name of the file.")
+  fileName?: string;
+
+  @doc("File type")
+  fileType?: FileType;
+
+  @doc("Expiry time of the file (ISO 8601 literal format)")
+  expireDateTime?: utcDateTime;
+
+  @doc("Validation status of the file")
+  validationStatus?: FileStatus;
+
+  @doc("Validation failure error details")
+  validationFailureDetails?: string;
+}
+
+@doc("The definition of an error object.")
+@error
+model ErrorResponseBody {
+  @doc("Error from a REST request.")
+  error: Error;
+
+  @header("x-ms-error-code")
+  @doc("The error code for specific error that occurred.")
+  errorCode?: string;
+}
+
+@doc("Error from a REST request.")
+model Error {
+  @doc("The error code.")
+  code: string;
+
+  @doc("The error message.")
+  message: string;
+
+  @doc("The error target.")
+  target?: string;
+
+  @doc("Additional details and inner errors.")
+  details?: Error[];
+}
+
+@doc("Collection of tests")
+@friendlyName("TestsList")
```

Comment:
I dont think you need `@friendlyName` here.

Reply:
Same comment in other places where `friendlyName` is being used.

Reply:
We added @friendlyName because the models were getting generated with the name similar to Paged{ResourceName} in this case PagedTest. In our existing spec we have name this model as TestsList. Should I remove this decorator and keep these models name to default?

Reply:
Ah I see, I think `@projectedName` would be better here. friendlyName is more for models that are templates, not ones that use templates. Here's the link to info on projected names: https://microsoft.github.io/typespec/standard-library/built-in-decorators#@projectedName

Reply:
I see this is still as friendlyName, are you planning on updating it to projectedName?

Reply:
@doc("The response to a metrics query.")
@projectedName("json", "Metrics")
//@friendlyName("Metrics")
model Metrics is Azure.Core.Page<TimeSeriesElement>;

tried this, but the json was still coming with PagedTimeSeriesElement. Although projectedName is working in case of parameter within models, as given in examples.

Reply:
@timotheeguerin just double checking before I resolve this comment, is friendlyName ok here?

Reply:
@Himanshu49 the issue is that the name isnt coming out as expected in the generated swagger correct? Have you tried generating an SDK with this to see what it outputs?

Reply:
This is an issue with `@friendlyname` we are discussing and why you shouldn't use `@friendlyname` in general but here yeah this is probably the only way to work around it right now.

Reply:
Thanks Tim! @Himanshu49 let's leave it with friendlyName for now if that's working for you

Reply:
This is the issue if you want to link it https://github.com/microsoft/typespec/issues/2276